README.rst

```diff
@@ -29,18 +33,18 @@ Setup
 Usage
 =====
 
-Locally copy an image from source to destination. Quay token is injected
+Locally copy an image from source to destination. Quay password is injected
 from the environment variable.
 ::
-  $ export AUTH_TOKEN=token
+  $ export QUAY_PASSWORD=token
```

Comment:
```suggestion
  $ export QUAY_PASSWORD=password
```

Reply:
Since the README examples use a robot account, what the user is inputting is a token, not a password